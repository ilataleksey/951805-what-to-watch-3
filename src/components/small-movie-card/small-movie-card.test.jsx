import React from "react";
import renderer from "react-test-renderer";
import SmallMovieCard from "./small-movie-card.jsx";
import film from "./small-movie-card.test.mocks.js";

it(`Render SmallMoviesCard`, () => {
  const tree = renderer.create(<SmallMovieCard
    film={film}
  />, {
    createNodeMock: () => {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
