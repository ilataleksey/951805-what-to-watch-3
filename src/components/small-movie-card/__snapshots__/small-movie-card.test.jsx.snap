// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render SmallMoviesCard 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Наименование фильма
    </a>
  </h3>
</article>
`;
